Lib/idlelib/idle_test/test_sidebar.py

```diff
@@ -370,5 +378,164 @@ def assert_colors_are_equal(colors):
         assert_colors_are_equal(orig_colors)
 
 
+class TestShellSidebar(unittest.TestCase):
```

Comment:
'IDLE Convention', Test at end:
```suggestion
class ShellSidebarTest(unittest.TestCase):
```